bam_readcount_helper.py

```diff
@@ -89,7 +89,13 @@ def filter_sites_in_hash(region_list, bam_file, ref_fasta, sample, output_dir, i
 if len(rc_for_snp.keys()) > 0:
     region_file = generate_region_list(rc_for_snp)
     filter_sites_in_hash(region_file, bam_file, ref_fasta, sample, output_dir, False)
+else:
+    output_file = os.path.join(output_dir, sample + '_bam_readcount_snv.tsv')
+    open(output_file, 'w').close()
```

Comment:
@susannasiebert can you take a look at this?

I'm a little worried about creating an empty file and silently running when there should have been output. Ex. Pindel in the GMS.

Reply:
it looks like the line `if len(rc_for_snp.keys()) > 0:` will ensure that we only produce an empty file when the input list is zero.